### PR TITLE
Fe css

### PR DESCRIPTION
**Pull request by Nguyephi**

### CODE REVIEW

**Review by azdoolittle — CHANGES_REQUESTED** [review]
You forgot nodemon haha

File: client/src/components/VideoPlayer.jsx [hunk 1]
```diff
@@ -3,15 +3,90 @@ import React from 'react';
...
+    constructor(props) {
+        super(props)
+        this.state = {
+            paused: false
+        }
+    }
+
+    play(e) {
+        const video = document.getElementById("video-player");
+        const playPause = document.getElementById("play-pause");
```
Comment (kraig-droid) [inline client/src/components/VideoPlayer.jsx:16]: Is this needed?

**Review by kraig-droid — APPROVED** [review]
lgtm

**Review by priscilagonzalez — COMMENTED** [review]
Lgtm.

**Review by kraig-droid — APPROVED** [review]
lgtm - this even helped me learn the github feature to look at only one commit instead of the whole pull -- yay!

**Pull request merged**